packages/grid/x-data-grid/src/components/toolbar/GridToolbarFilterButton.tsx

```diff
@@ -119,7 +119,6 @@ const GridToolbarFilterButton = React.forwardRef<HTMLButtonElement, GridToolbarF
         <rootProps.components.BaseButton
           ref={ref}
           size="small"
-          color="primary"
```

Comment:
color primary is now the default in the minimum version of Material UI we depend on: https://mui.com/material-ui/migration/v5-component-changes/#%E2%9C%85-remove-default-color-prop